templates/exception.php

```diff
@@ -9,515 +9,694 @@
 <html lang="en">
 
 <head>
-    <meta charset="utf-8"/>
-    <title><?= $this->htmlEncode($this->getThrowableName($throwable)) ?></title>
+    <meta charset="utf-8" />
+    <title>
+        <?= $this->htmlEncode($this->getThrowableName($throwable)) ?>
+    </title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700">
```

Comment:
I'd use local font instead. Yii may be used in local environments w/o Internet connection.

Reply:
How can we use this font locally if we don't include it in public/assets or public/vendor/assets?

Reply:
You're right. That would complicate things a bit too much.